### PR TITLE
Added option to zoom document with the use of ctrl+scroll

### PR DESCRIPTION
**Pull request by PrimosK**
This PR adds convenient `ctrl+scroll` shortcut to zoom in/out documents.

Currently the effect of `ctr+scroll` is exactly the same as changing a zoom slider. A future improvement might be to actually zoom at the point where mouse pointer is located at the time when zoom action is initiated.

Side note:
That's all I wanted to contribute (for now). Having a new micro release in the near future would be great and thanks for looking after my PRs in timely manner.

### CODE REVIEW

File: gemsfx/src/main/java/com/dlsc/gemsfx/skins/PDFViewSkin.java [hunk 1]
```diff
@@ -681,6 +681,17 @@ private void ensureVisible(Node node) {
             wrapper.setMaxWidth(Region.USE_PREF_SIZE);
             wrapper.setMaxHeight(Region.USE_PREF_SIZE);
             wrapper.rotateProperty().bind(pdfView.pageRotationProperty());
+            wrapper.addEventHandler(ScrollEvent.SCROLL, evt -> {
+                if (evt.isControlDown()) {
```
Comment (dlemmermann) [inline gemsfx/src/main/java/com/dlsc/gemsfx/skins/PDFViewSkin.java:685]: Can you please replace this with eve.isShortcutDown()? On Mac CTRL + Scroll zooms the whole desktop (if enabled in system preferences).

File: gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java [hunk 2]
```diff
@@ -284,6 +284,35 @@ public final boolean gotoLastPage() {
...
     }
 
+    /**
+     * Convenience method to decrease the zoom factor for value specified as {@code delta}.
+     *
+     * @param delta zoom factor (decrese) delta
+     * @return true if the operation actually did cause a zoom change
+     */
+
+    public final boolean decreaseZoomFactor(double delta) {
```
Comment (dlemmermann) [inline gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java:294]: These two methods are utility methods that need to go into the skin. JavaFX controls usually never expose doThis(), doThat() convenience methods on the controls themselves but only provide properties.

File: gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java [hunk 3]
```diff
@@ -284,6 +284,35 @@ public final boolean gotoLastPage() {
         return currentPage != getPage();
     }
 
+    /**
+     * Convenience method to decrease the zoom factor for value specified as {@code delta}.
+     *
+     * @param delta zoom factor (decrese) delta
```
Comment (dlemmermann) [inline gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java:290]: spelling: "decrees" -> "decrease".

File: gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java (hunk 2 above)
Comment (PrimosK) [reply gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java]: Ack.

File: gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java (hunk 3 above)
Comment (PrimosK) [reply gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java]: Ack.

**Pull request merged**